cue/scuemata/panel-plugin.cue

```diff
@@ -8,10 +8,10 @@ package scuemata
 // the larger Dashboard schema.
 #PanelSchema: {
     // Defines plugin specific options for a panel
-    PanelOptions: {...}
+    PanelOptions: {...} @cuetsy(kind="interface")
```

Comment:
it is not top level, i wonder in the future we might need to go lower in cuetsy :)

Reply:
That part is OK - we can reasonably expect the caller to be responsible for passing in a `cue.Value` where those attributes are top level.